eZ/Publish/Core/Search/Legacy/Content/Common/Gateway/CriterionHandler/FullText.php

```diff
@@ -178,14 +178,21 @@ protected function getWordExpression(SelectQuery $query, $token)
      * @param \eZ\Publish\Core\Persistence\Database\SelectQuery $query
      * @param string $string
      *
-     * @return \eZ\Publish\Core\Persistence\Database\SelectQuery
+     * @return \eZ\Publish\Core\Persistence\Database\SelectQuery|null
```

Comment:
*Also note that `handle()` should add phpdoc that it uses `getWordIdSubquery` so the throw part gets reflected there to.